libs/micronaut-newrelic/src/main/java/com/agorapulse/micronaut/newrelic/FallbackNewRelicInsightsService.java

```diff
@@ -48,7 +53,8 @@ public <E> void unsafeCreateEvents(@NonNull @Valid Collection<E> events) throws
     @Override
     public <E> void createEvents(Collection<E> events) {
         try {
-            LOGGER.info("Following events not sent to NewRelic:\n{}", mapper.writerWithDefaultPrettyPrinter().writeValueAsString(events));
+            List<Map<String, Object>> payloads = events.stream().map(extractor::extractPayload).toList();
```

Comment:
wouldn't it be much easier simply document that pople can use Jackson to customize the event payload?

Reply:
I don't understand, using Jackson is what I did [here](https://github.com/agorapulse/platform-commons/blob/master/micronaut-libs/commons-exceptions-newrelic/src/main/java/agorapulse/commons/exceptions/newrelic/NewRelicErrorEvent.java#L99) but it does not work except for fallback service.
Jackson is not used in actual implementations

Reply:
ah ok, haven't noticed this is the fallback service